recipes/scalene/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "scalene" %}
+{% set version = "1.5.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scalene-{{ version }}.tar.gz
+  sha256: a3adfe397e1e379091e3405c692a178291ebe8c2d3721dbc02bfec5ecc347ef7
+
+build:
+  number: 0
+  skip: true   # [py<38]
+  entry_points:
+    - scalene = scalene.__main__:main
+  script: make bdist #{{ PYTHON }} -m pip install . -vv
```

I'm going to go out on a limb and say that we don't need to `make` a `bdist`.

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```